system-test/bigtable.ts

```diff
@@ -176,6 +176,85 @@ describe('Bigtable', () => {
     });
   });
 
+  describe('CMEK', () => {
+    let kmsKeyName: string;
+
+    const CMEK_INSTANCE = bigtable.instance(generateId('instance'));
+    const CMEK_CLUSTER = CMEK_INSTANCE.cluster(generateId('cluster'));
+
+    before(async () => {
+      const projectId = await bigtable.auth.getProjectId();
+      const cryptoKeyId = generateId('key');
+      const keyRingId = generateId('key');
+      const keyRingsBaseUrl = `https://cloudkms.googleapis.com/v1/projects/${projectId}/locations/us-central1/keyRings`;
+      kmsKeyName = `projects/${projectId}/locations/us-central1/keyRings/${keyRingId}/cryptoKeys/${cryptoKeyId}`;
+
+      await bigtable.auth.request({
```

Comment:
is this creating a key per test run? If so, I think these will need to be cleaned up as well?

Reply:
Good catch, it was. I added a commit that uses the [`destroy`](https://cloud.google.com/kms/docs/reference/rest/v1/projects.locations.keyRings.cryptoKeys.cryptoKeyVersions/destroy) method to schedule the destroy of the key. Please let me know if there's a better or more thorough way to remove these crypto key resources!